ejb/WFLY-6143-Ability-to-add-a-server-side-configured-Interceptor-for-the-applications-without-changing-the-deployments.adoc

```diff
@@ -0,0 +1,129 @@
+= Ability to add a server-side configured Interceptor for the applications without changing the deployments
+:author:            Tomasz Adamski
+:email:             tomasz.adamski@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+The goal of this issue is to implement server side EJB interceptors that can be configured system-wide through ejb3 subsystem configuration.
+
+=== Issue
+* https://issues.jboss.org/browse/WFLY-6143
+
+=== Dev Contacts
+
+* mailto:tadamski@redhat.com
+
+=== QE Contacts
+
+=== Testing By
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+EJB
+
+=== WildFLY custom interceptors
+
+WildFly allows users to implement their own intereceptors as a part of the deployment (https://docs.jboss.org/author/display/WFLY10/Container+interceptors). Most important characteristics of those are as follows:
```

Comment:
Please use https://docs.wildfly.org/17/Developer_Guide.html#Container_interceptors as the docs URL.